### PR TITLE
HSD8-572 Style login form to match previous

### PR DESCRIPTION
**Pull request by pookmish**
# READY FOR REVIEW

# Summary
- Style login form to match what it was with material

# Needed By (Date)
- 2/13

# Urgency
- Low

# Steps to Test
1. Checkout this branch
1. using any site, enable stanford_ssp `drush en stanford_ssp`
1. view `/user/login` page to validate it looks like current production sites.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: docroot/themes/humsci/su_humsci_admin/Gruntfile.js [hunk 1]
```diff
@@ -102,13 +102,17 @@ module.exports = function(grunt) {
...
       },
       dist: {
-        files: {
-          // Files are compiled individually so they may be included
-          // conditionally using logic built in to the theme template or module.
-
-          // BASE
-          'css/base/base.css':    'scss/base/base.scss',
-        }
+        files: [{
```
Comment (sherakama) [inline docroot/themes/humsci/su_humsci_admin/Gruntfile.js:105]: Are you sure you want this change in here? This seems outside of the scope of the PR.

File: docroot/themes/humsci/su_humsci_admin/package-lock.json [hunk 2]
```diff
@@ -7,7 +7,7 @@
     "abbrev": {
```
Comment (sherakama) [inline docroot/themes/humsci/su_humsci_admin/package-lock.json:7]: Are you sure you want changes to this file in this PR? This seems outside of the scope of the PR.

**Review by sherakama — CHANGES_REQUESTED** [review]
There was a bunch of change in the Gruntfile.js and package-lock.json that I am not sure you intended on having in the PR. Can you confirm that you want those changes and/or remove them.

File: docroot/themes/humsci/su_humsci_admin/package-lock.json (hunk 2 above)
Comment (pookmish) [reply docroot/themes/humsci/su_humsci_admin/package-lock.json:7]: i'm guessing that was injected by npm during `npm install` and the latest npm version

File: docroot/themes/humsci/su_humsci_admin/Gruntfile.js (hunk 1 above)
Comment (pookmish) [reply docroot/themes/humsci/su_humsci_admin/Gruntfile.js:105]: yeah. allowed a dynamic scss compiling since the scss files that needed be modified were not compiling

**Review by sherakama — APPROVED** [review]
Good by me. @josephgknox you're up.

**Review by josephgknox — APPROVED** [review]
Looks like it should to me, too.
